app/js/components/labOrderEntry/DraftOrder/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import DraftList from './DraftList';
+
+export class LabDraftOrder extends React.Component {
+  render() {
+    const { draftLabOrders, disableCancelButton, disableSaveButton } = this.props;
+    const lengthOfDraftLabOrders = draftLabOrders.length;
+    return (
+      <div className="draft-spacing draft-lab-layout">
+        <h5 className="h5-draft-header">
+          Unsaved Draft Orders ({lengthOfDraftLabOrders})
+        </h5>
+        <div className="table-container">
+          <ul className="draft-list-container">
+            <DraftList
+              draftOrders={draftLabOrders}
+            />
+          </ul>
+        </div>
+        <br />
+        <input
+          type="button"
+          onClick={() => {}}
+          className="button cancel modified-btn"
+          value={lengthOfDraftLabOrders > 1 ? "Discard All" : "Discard"}
+          disabled={disableCancelButton}
+        />
+        <input
+          type="submit"
+          onClick={() => {}}
+          className="button confirm right modified-btn"
+          value="Sign and Save"
+          disabled={disableSaveButton}
+        />
+      </div>
+    );
+  }
+}
+
+LabDraftOrder.propTypes = {
+  draftLabOrders: PropTypes.arrayOf(PropTypes.any).isRequired,
+  disableCancelButton: PropTypes.bool.isRequired,
+  disableSaveButton: PropTypes.bool.isRequired,
+};
+
+const mapStateToProps = ({
+  draftLabOrderReducer: { draftLabOrders },
+}) => ({
+  draftLabOrders,
+});
+
+export default connect(mapStateToProps)(LabDraftOrder);
```

Comment:
I believe the component where this is exported to is connected to redux, so you could try to make this more as a presentational component sorta or even stateless or purecomponent even

Reply:
Thanks, the connect method has been taken away and the component is now a stateless one.